functions.cmake

```diff
@@ -86,21 +86,17 @@ function(preprocess_def_file inputFilename outputFilename)
                               PROPERTIES GENERATED TRUE)
 endfunction()
 
-function(generate_exports_file)
-  set(INPUT_LIST ${ARGN})
-  list(GET INPUT_LIST -1 outputFilename)
-  list(REMOVE_AT INPUT_LIST -1)
-
+function(generate_exports_file INPUT_FILE OUTPUT_FILE)
```

Comment:
A nit - similar functions in this file use inputFilename / outputFilename naming of the arguments, could you please change the names here to match those?